app/assets/stylesheets/editor.css

```diff
@@ -9,7 +9,9 @@
   margin-bottom:10px;
   background:white;
 }
-
+.cursor-dropzone{
+  cursor: pointer;
+}
```

Comment:
Hey @ValeriaCobianISW, please remove the styling as you have used inline styling now. Thanks!

Reply:
Ready :D thanks @gauravano

Reply:
Great!